superset-frontend/src/chart/ChartRenderer.jsx

```diff
@@ -78,28 +119,35 @@ class ChartRenderer extends React.Component {
   }
 
   shouldComponentUpdate(nextProps) {
-    const resultsReady =
-      nextProps.queryResponse &&
-      ['success', 'rendered'].indexOf(nextProps.chartStatus) > -1 &&
-      !nextProps.queryResponse.error &&
-      !nextProps.refreshOverlayVisible;
-
-    if (resultsReady) {
-      this.hasQueryResponseChange =
-        nextProps.queryResponse !== this.props.queryResponse;
-
-      if (
-        this.hasQueryResponseChange ||
-        nextProps.annotationData !== this.props.annotationData ||
-        nextProps.height !== this.props.height ||
-        nextProps.width !== this.props.width ||
-        nextProps.triggerRender ||
-        nextProps.formData.color_scheme !== this.props.formData.color_scheme
-      ) {
-        return true;
-      }
+    // if no results loaded, don't render
+    if (
+      !nextProps.queryResponse ||
+      nextProps.queryResponse.error ||
+      nextProps.refreshOverlayVisible
+    )
+      return false;
```

Comment:
add `{}`. Not sure why `eslint` is not warning.

Reply:
I think both are acceptable. Would actually prefer no `{}` if the conditions are simpler.